indico_themes_canonical/static/js/ubuntu_summit.js

```diff
@@ -115,7 +115,7 @@ function buildHeaderStrip() {
   headerLogoCont.classList.add("headerLogoCont");
   const headerLogo = document.createElement("img");
   headerLogo.classList.add("headerLogo");
-  headerLogo.src = "https://assets.ubuntu.com/v1/88af8658-ubuntu+summit-illustration.svg"
+  headerLogo.src = "https://assets.ubuntu.com/v1/5a965c5c-summit-2023-logo.png"
```

Comment:
This change will still change the Ubuntu Summit 2022 logo to the 2023 one. Was it intended to merge this, or was the main purpose fixing the error message colors?